### PR TITLE
Add function to check the output of any cmds

### PR DESCRIPTION
**Pull request by yafu-1**
Signed-off-by: Yan Fu <yafu@redhat.com>

### CODE REVIEW

**Comment by yafu-1** [comment]
@dzhengfy Could you help to review the pr please? Thanks.

File: virttest/utils_test/libvirt.py [hunk 1]
```diff
@@ -3619,6 +3619,26 @@ def check_qemu_cmd_line(content, err_ignore=False):
...
+    Check the specified content in the output of the cmd
+    :param cmd: the cmd wants to check
+    :param content: the desired string to search
+    :param err_ignore: True to return False when fail
+                       False to raise exception when fail
+    :return: True if exist, False otherwise
+    """
+    if cmd is None:
+        raise exceptions.TestFail("cmd can not be None")
+    cmd_output = results_stdout_52lts(process.run(cmd, shell=True)).strip()
```
Comment (dzhengfy) [inline virttest/utils_test/libvirt.py:3633]: results_stdout_52lts is going to be removed. Please use  process.run(xxx).stdout_text.strip()

File: virttest/utils_test/libvirt.py [hunk 2]
```diff
@@ -3619,6 +3619,26 @@ def check_qemu_cmd_line(content, err_ignore=False):
...
 
 
+def check_cmd_output(cmd, content, err_ignore=False):
+    """
+    Check the specified content in the output of the cmd
+    :param cmd: the cmd wants to check
+    :param content: the desired string to search
+    :param err_ignore: True to return False when fail
+                       False to raise exception when fail
+    :return: True if exist, False otherwise
```
Comment (dzhengfy) [inline virttest/utils_test/libvirt.py:3629]: Add :raise: docstring

File: virttest/utils_test/libvirt.py (hunk 1 above)
Comment (dzhengfy) [inline virttest/utils_test/libvirt.py:3633]: process.run(cmd, shell=True) will raise an exception anyway if fail. Do you want to pass 'err_ignore' to process.run()?

Comment (yafu-1) [reply virttest/utils_test/libvirt.py]: updated.

File: virttest/utils_test/libvirt.py (hunk 2 above)
Comment (yafu-1) [reply virttest/utils_test/libvirt.py:3629]: updated.

File: virttest/utils_test/libvirt.py (hunk 1 above)
Comment (yafu-1) [reply virttest/utils_test/libvirt.py]: updated.

File: virttest/utils_test/libvirt.py (hunk 2 above)
Comment (dzhengfy) [reply virttest/utils_test/libvirt.py:3629]: @yafu-1 , sorry to make you misunderstand. What I mean is to add a docstring for :raise:. 
For example:
:raise: exceptions.TestFail when xxxx

File: virttest/utils_test/libvirt.py [hunk 3]
```diff
@@ -3619,6 +3619,28 @@ def check_qemu_cmd_line(content, err_ignore=False):
...
+    :raise: docstring
+    """
+    if cmd is None:
+        raise exceptions.TestFail("cmd can not be None")
+        cmd_output = process.run(cmd, shell=True,
```
Comment (dzhengfy) [inline virttest/utils_test/libvirt.py:3634]: This line will not be executed. Please fix the indention.

Comment (yafu-1) [reply virttest/utils_test/libvirt.py]: Updated.

Comment (yafu-1) [reply virttest/utils_test/libvirt.py]: Updated.

File: virttest/utils_test/libvirt.py [hunk 4]
```diff
@@ -3619,6 +3619,28 @@ def check_qemu_cmd_line(content, err_ignore=False):
...
 
+def check_cmd_output(cmd, content, err_ignore=False):
+    """
+    Check the specified content in the output of the cmd
+    :param cmd: the cmd wants to check
+    :param content: the desired string to search
+    :param err_ignore: True to return False when fail
+                       False to raise exception when fail
+    :return: True if exist, False otherwise
+    :raise: exceptions, TestFail when content not found in the output of cmd
```
Comment (dzhengfy) [inline virttest/utils_test/libvirt.py:3630]: Minor typo fix.
exceptions, TestFail => exceptions.TestFail when content is not found

File: virttest/utils_test/libvirt.py [hunk 5]
```diff
@@ -3619,6 +3619,28 @@ def check_qemu_cmd_line(content, err_ignore=False):
     return True
 
 
+def check_cmd_output(cmd, content, err_ignore=False):
```
Comment (chunfuwen) [inline virttest/utils_test/libvirt.py:3622]: suggest provide one more parameter :session, which can wrap VM or remote host scenarios.

Comment (yafu-1) [reply virttest/utils_test/libvirt.py]: Updated.

**Pull request merged**